401 Challenges/challenge01/01.README.md

```diff
@@ -0,0 +1,94 @@
+# Reverse an array.
+
+Reverse an array such that the contents at the end is now at the beginning and vice versa.
+
+## Whiteboard Process
+- Notability ([direct link to image](https://notability.com/n/0OZm7qFi4EpdclH08EkBBu))
+- Images
+![Page1](image.png)
+![Page2](image-1.png)
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+- I first paraphrased the question
+- Then I sketched out what would need to happen
+- Then I spoke about what needed to happen
+- I noticed the spec said not to use built in functions, so I didn't assume we'd know the length of the array
```

Comment:
Great job Paul! The problem did say you couldn't use any built in methods but .length is considered a property so you could have certainly used it. But way to go figuring out how to get to the solution without even using the length!